src/papi_events.csv

```diff
@@ -1864,6 +1864,52 @@ PRESET,PAPI_BR_UCN,NOT_DERIVED,PM_BR_UNCOND
 PRESET,PAPI_BR_TKN,NOT_DERIVED,PM_BR_CORECT_PRED_TAKEN_CMPL
 PRESET,PAPI_FXU_IDL,NOT_DERIVED,PM_FXU_IDLE
 #
+CPU,POWER10
+CPU,power10
+#
+PRESET,PAPI_L1_DCM,DERIVED_ADD,PM_DATA_FROM_L1MISS,PM_ST_MISS_L1
+PRESET,PAPI_L1_LDM,NOT_DERIVED,PM_DATA_FROM_L1MISS
+PRESET,PAPI_L1_STM,NOT_DERIVED,PM_ST_MISS_L1
+PRESET,PAPI_L1_DCW,DERIVED_SUB,PM_ST_FIN,PM_ST_MISS_L1
+PRESET,PAPI_L1_DCR,NOT_DERIVED,PM_LD_HIT_L1
+PRESET,PAPI_L1_DCA,DERIVED_ADD,PM_LD_REF_L1,PM_ST_CMPL
+PRESET,PAPI_L2_DCM,DERIVED_ADD,PM_DATA_FROM_L2MISS,PM_L2_ST_MISS
+PRESET,PAPI_L2_LDM,NOT_DERIVED,PM_L2_LD_MISS
+PRESET,PAPI_L2_STM,NOT_DERIVED,PM_L2_ST_MISS
+PRESET,PAPI_L2_DCR,NOT_DERIVED,PM_DATA_FROM_L2
+PRESET,PAPI_L2_DCW,NOT_DERIVED,PM_L2_ST_HIT
+PRESET,PAPI_L3_DCR,NOT_DERIVED,PM_DATA_FROM_L3
+PRESET,PAPI_L3_DCM,NOT_DERIVED,PM_DATA_FROM_L3MISS
+PRESET,PAPI_L3_LDM,NOT_DERIVED,PM_DATA_FROM_L3MISS
+PRESET,PAPI_L1_ICH,NOT_DERIVED,PM_INST_FROM_L1
```

Comment:
The PM_INST_FROM_L1 native event seems to overcount L1I hits.

Reply:
PM_INST_FROM_L1 will count all L1I hits that's the assumption.